smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -334,6 +339,64 @@ protected async Task VerifyDataOnIoTHub(string moduleId)
             await eventHubClient.CloseAsync();
         }
 
+        protected async Task VerifyTwinAsync()
+        {
+            if (this.TwinTestFileName.HasValue)
+            {
+                string twinTestJson = JObject.Parse(File.ReadAllText(this.TwinTestFileName.GetOrElse(string.Empty))).ToString();
+
+                var twinTest = JsonConvert.DeserializeObject<TwinTestConfiguration>(twinTestJson);
+
+                Twin currentTwin = await this.context.RegistryManager.GetTwinAsync(this.context.Device.Id, twinTest.ModuleId);
+
+                if (twinTest.TwinTest.ContainsKey("desired") && twinTest.TwinTest["desired"].Count > 0)
+                {
+                    StringBuilder twinpatch = new StringBuilder(@"{ properties: { desired: { ");
```

Comment:
String manipulation here is not a good idea - we should add the entire desired/reported properties in our json file and set the desired properties json object as it is..

Reply:
Let's sync up. I think the problem with that is that, on the Desired Property we have other fields (Like Metadata) that I don't want to compare here. The test here would just in our fields, making sure the Key/Value Pair we se on desired, shows up (with the correct version) on reported property.

Reply:
Done! Thanks a lot! Cleaned a bunch!